float32 lin_vel_ref
float32 ang_vel_ref
